tests/test_admin.py

```diff
@@ -177,6 +178,34 @@ def test_change_form_compliance_number_field_is_not_editable_in_unpublished_vers
 
         self.assertIn(unpublished_expected_response, decoded_response)
 
+    def test_change_form_fieldset_ordering_draft(self):
+        """
+        The change form should remain in same order, starting with compliance number when in draft
+        """
+        content_expiry = PollContentExpiryFactory(version__state=DRAFT)
+        endpoint = self.get_admin_url(ContentExpiry, "change", content_expiry.pk)
+        expected_fieldset = CONTENT_EXPIRY_FIELDSETS
+        with self.login_user_context(self.get_superuser()):
```

Comment:
```suggestion

        with self.login_user_context(self.get_superuser()):
```

Needs a break between the Arrange and Act